v2/internal/config/vars.go

```diff
@@ -14,8 +14,11 @@ import (
 )
 
 const (
+	// #nosec
+	AzureClientSecret          = "AZURE_CLIENT_SECRET"
 	SubscriptionIDVar          = "AZURE_SUBSCRIPTION_ID"
 	TenantIDVar                = "AZURE_TENANT_ID"
+	AzureClientID              = "AZURE_CLIENT_ID"
```

Comment:
minor: put `var` suffix on the end of these, to indicate they're the value of the variable name, not the actual secret or client id themselves? (Follows pattern of other names here)